data/adapters/file_fileset_adapter.py

```diff
@@ -69,24 +94,26 @@ def get_software_igvf(self, file_object, portal_url):
                     portal_url + software_version + '/@@object?format=json').json()
                 software_object = requests.get(
                     portal_url + software_version_object['software'] + '/@@object?format=json').json()
-                software.add(software_object['name'])
+                software.add(software_object['title'])
         return software
 
-    def parse_annotation(self, dataset_object, portal_url, prediction, prediction_method, software, preferred_assay_titles, assay_term_ids):
+    def parse_annotation(self, dataset_object, portal_url, class_type, method, software, preferred_assay_titles, assay_term_ids):
+        method = dataset_object['annotation_type']
         if 'prediction' in dataset_object['annotation_type']:
```

Comment:
this approach works most cases, but need to revisit manually curated caQTLs (ENCFF103XRK and ENCFF776CZZ), they should be 'experimental'.